corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -724,7 +721,7 @@ def report_column_options(self):
         # NOTE: Count columns aren't useful for table reports. But we need it in the column options because
         # the options are currently static, after loading the report builder a user can switch to an aggregated
         # report.
-        count_col = CountColumn("Number of Cases" if self.source_type == "case" else "Number of Forms")
+        count_col = CountColumn("number_of_cases" if self.source_type == "case" else "number_of_forms")
```

Comment:
why this change?

Reply:
According to the spec: https://docs.google.com/document/d/1XE-gUOcpRyxOQXRSQkEzI38q9JY5qwm5mj70vVV5vgM/edit?ts=5fbd21a9#bookmark=id.b6ah57k2m04x

They asked to change all properties to match the case.

Reply:
Hey @proteusvacuum can you confirm if this response is sufficient? I'm just going through this PR to figure out which comments need to be resolved.

Reply:
This code change was removed, marking conversation resolved.